string_lab.js

```diff
@@ -0,0 +1,65 @@
+//#1 console.log(); the first character of a string
+ let name = "ohidur"
+ console.log(name[0]);
+//#2 log the lenth of the string
+
+console.log(name.length);
+//#3 log the last character of any string.
+
+console.log(name[name.length-1]);
+
+/*
+4. Create a code block that takes a single string variable, and returns a copy of the string with the last letter capitalized. For example:
```

Comment:
The question was asking for last letter to be capitalized. What you wrote below would not work. How can we solve this problem using two methods: slice() and toUpperCase().